rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -855,6 +855,11 @@ public static void setTranportPoolMinSize(Configuration conf, int count)
     conf.setInt(KEY_POOL_MIN_SIZE, count);
   }
 
+  public static void setTranportPoolMaxSize(Configuration conf, int count)
```

Comment:
I created getters with name like 'getPoolSizeMax', can we also change those?

Reply:
changing it to `getTranportPoolMaxSize`